### PR TITLE
Implement automated build of docker images on github

### PR DESCRIPTION
**Pull request by GreatAlexander**

### CODE REVIEW

**Comment by GreatAlexander** [comment]
pre-commit.ci autofix

**Comment by GreatAlexander** [comment]
pre-commit.ci autofix

File: dev.sh [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
 # ----------------------------------------------------------------
-# Build docker dev stage and add local code for live development 
+# Build docker dev stage and add local code for live development
 # ----------------------------------------------------------------
 
 # Build docker image up to dev stage
 DOCKER_BUILDKIT=1 docker build \
--t imu_humble \
--f Dockerfile --target dev .
+    -t imu_humble \
```
Comment (hect95) [inline dev.sh]: Please change this for `av_imu:latest`

File: runtime.sh [hunk 2]
```diff
@@ -6,18 +6,18 @@
...
+# If an arg is defined, start container with bash
 if [ -n "$1" ]; then
     CMD="bash"
 fi
 
 # Build docker image only up to base stage
 DOCKER_BUILDKIT=1 docker build \
--t imu_humble \
--f Dockerfile --target runtime .
+    -t imu_humble \
```
Comment (hect95) [inline runtime.sh]: Please change this for `av_imu:latest`

**Review by hect95 — DISMISSED** [review]
Minor changes

File: .github/workflows/docker.yml [hunk 3]
```diff
@@ -0,0 +1,53 @@
+name: Create and publish a Docker image
+
+on:
+  push:
+    tags: ["*"]
+
+env:
+  REGISTRY: ghcr.io
+  IMAGE_NAME: ${{ github.repository }}
```
Comment (hect95) [inline .github/workflows/docker.yml:9]: I guess here we would need to adapt the docker name?

Comment (hect95) [reply .github/workflows/docker.yml:9]: I guess it would be: `IMAGE_NAME: av_imu`

Comment (GreatAlexander) [reply .github/workflows/docker.yml:9]: Renamed github repo so this variable value should now be as expected: `av_imu`

File: runtime.sh (hunk 2 above)
Comment (GreatAlexander) [reply runtime.sh]: Done

File: dev.sh (hunk 1 above)
Comment (GreatAlexander) [reply dev.sh]: Done

**Review by hect95 — APPROVED** [review]
Ready to merge!

**Pull request merged**